### PR TITLE
Simplify README and consolidate scripts

### PR DESCRIPTION
**Pull request by benthorner**
Partly addresses: https://github.com/alphagov/notifications-manuals/issues/9

Please see the commit messages for more details.

---

🚨⚠️ This will be deployed automatically all the way to production when you click merge ⚠️🚨

For more information, including how to check this deployment on preview or staging first before it goes to production, see our [team wiki section on continuous deployment](https://github.com/alphagov/notifications-manuals/wiki/Deploying-with-concourse#continuous-deployment)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,22 +14,18 @@ help:
 	@cat $(MAKEFILE_LIST) | grep -E '^[a-zA-Z_-]+:.*?## .*$$' | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 .PHONY: run
```
Comment (klssmith) [inline Makefile:16]:
```suggestion
.PHONY: run-flask
```

File: Makefile [hunk 2]
```diff
@@ -17,22 +14,18 @@ help:
...
 
 .PHONY: test
-test:
-	find . -name \*.pyc -delete
-	npm install
-	npm run build
+test: test-requirements
 	./scripts/run_tests.sh
 
 .PHONY: build
```
Comment (klssmith) [inline Makefile:24]:
```suggestion
.PHONY: bootstrap
```

**Review by klssmith — APPROVED** [review]
Left a really minor suggestion, but it all worked well for me locally 👍

File: Makefile (hunk 1 above)
Comment (benthorner) [reply Makefile]: Nice catch! I've fixed this and the other `build` / `bootstrap` discrepancy 👍.

**Pull request merged**